monkey/monkey_island/cc/resources/ip_addresses.py

```diff
@@ -0,0 +1,18 @@
+from monkey_island.cc.resources.AbstractResource import AbstractResource
+from monkey_island.cc.resources.request_authentication import jwt_required
+from monkey_island.cc.services.utils.network_utils import local_ip_addresses
+
+
+class IpAddresses(AbstractResource):
+    urls = ["/api/island/ip-addresses"]
+
+    @jwt_required
+    def get(self):
+        """
+        Gets the local ip address from the Island
+
+        :return: a list of local ips
+        """
+        local_ips = local_ip_addresses()
+
+        return {"ip_addresses": local_ips}
```

Comment:
```suggestion
        return local_ips
```

Since this is a collection, we can just return a list.

Reply:
We can't do that without changes to `monkey/monkey_island/cc/services/representations.py:normalize_object`

Reply:
We'll have to fix that at some point, but now probably isn't the time. The `:return:` in the docstring is wrong, though.